app/assets/stylesheets/components.scss

```diff
@@ -243,6 +243,25 @@
   max-width: 100%;
 }
 
+.form-group {
+
+  .field_with_errors {
+    width: 100%;
+  }
+}
+
+.form-input {
```

Comment:
@MauryanKansara could you solve this issue?

Reply:
> @MauryanKansara could you solve this issue?

I am not able to understand what this bot is trying to say. I don't know what rule it is talking about.

Reply:
@MauryanKansara It is stating that in line number 239, if you see in the editor, there is already `.form-input` defined, you just need to merge this line of code there to prevent duplication on class properties.

Reply:
@vedant-jain03 got it! I will make the changes and push it. Thank you for helping me out!

Reply:
@vedant-jain03 I made this [commit](https://github.com/CircuitVerse/CircuitVerse/pull/3570/commits/45414f2686d97eef2a89fd4975726a2ef7311b32) accidentally, I tried to delete it but it seems to be not working. So is it okay to keep this commit? Sorry for the trouble I've caused.

Reply:
No worries, you can write style-lint comment to ignore that!

Reply:
@vedant-jain03 really sorry for the late reply, I used style-lint comment to ignore that error and pushed the changes, but it seemed to not working. Should I ignore them? Cuz the issue is already fixed.